README.md

```diff
@@ -176,14 +176,73 @@ let nftType = nft.getType()
 
 The [example NFT contract](contracts/ExampleNFT.cdc) shows how to implement metadata views.
 
-### List of common views
+### List of views
+
+| Name       | Purpose                                    | Status      | Source                                                                                                   |
+| ----------- | ------------------------------------------ | ----------- | -------------------------------------------------------------------------------------------------------- |
+| `Display`   | Return the basic representation of an NFT. | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L31-L66)  |
+| `HTTPFile`  | A file available at an HTTP(S) URL.        | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L86-L98)  |
+| `IPFSFile`  | A file stored in IPFS.                     | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L100-L137) |
+| `Edition`   | Return information about one or more editions for an NFT. | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L139-L174) |
+| `Editions`  | Wrapper for multiple edition views.        | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L176-L187)|
+| `Serial`    |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L200-L211)|
```

Comment:
It looks like some of these are missing descriptions. Can you add some for the ones that are missing?